mbtiles/src/patcher.rs

```diff
@@ -169,42 +169,59 @@ mod tests {
     use crate::MbtilesCopier;
 
     #[actix_rt::test]
-    async fn apply_flat_patch_file() -> MbtResult<()> {
+    async fn apply_flat_patch_file() {
         // Copy the src file to an in-memory DB
-        let src_file = PathBuf::from("../tests/fixtures/mbtiles/world_cities.mbtiles");
-        let src = PathBuf::from("file:apply_flat_diff_file_mem_db?mode=memory&cache=shared");
+        let src_file = PathBuf::from("file:flat_src_file_mem?mode=memory&cache=shared");
+        let mbt = Mbtiles::new(&src_file).unwrap();
+        let mut conn = mbt.open().await.unwrap();
+        let script = std::fs::read_to_string("../tests/fixtures/mbtiles/world_cities.sql").unwrap();
+        sqlx::raw_sql(&script).execute(&mut conn).await.unwrap();
+        let src = PathBuf::from("file:apply_flat_patch_file?mode=memory&cache=shared");
 
         let mut src_conn = MbtilesCopier {
             src_file: src_file.clone(),
             dst_file: src.clone(),
             ..Default::default()
         }
         .run()
-        .await?;
+        .await
+        .unwrap();
 
         // Apply patch to the src data in in-memory DB
-        let patch_file = PathBuf::from("../tests/fixtures/mbtiles/world_cities_diff.mbtiles");
-        apply_patch(src, patch_file, true).await?;
+        let patch_file = PathBuf::from("file:flat_patch_file_mem?mode=memory&cache=shared");
+        let mbt = Mbtiles::new(&patch_file).unwrap();
+        let mut conn = mbt.open().await.unwrap();
+        let script =
+            std::fs::read_to_string("../tests/fixtures/mbtiles/world_cities_diff.sql").unwrap();
```

Comment:
i would just use `include_str!("...")` and elsewhere - it's better for a test to focus on the testing, not to any extras like file IO